pages/events/index.vue

```diff
@@ -31,19 +31,32 @@ import QiskitPage from '~/components/qiskit/QiskitPage.vue'
 
 export default class extends QiskitPage {
   routeName: string = 'events'
+  windowWidth: Number = 0
 
   autoplayVideo () {
+    if (!this.$refs.video) {
+      return
+    }
+
+    const video = this.$refs.video as any
+
     setTimeout(() => {
-      if (this.$refs.video) {
-        this.$refs.video.load()
-        this.$refs.video.muted = true
-        this.$refs.video.play()
-      }
+      video.load()
+      video.muted = true
+      video.play()
     }, 1)
   }
 
-  created () {
-    this.autoplayVideo()
+  get isDesktop () {
+    return this.windowWidth > 600
```

Comment:
Why not using `window.innerWidth` directly?

Reply:
Because I couldn't access `window` in a computed property directly. It's only available in the `mounted` object.